jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/HttpFieldsTest.java

```diff
@@ -1000,13 +1000,90 @@ public void testPutNullName()
         assertThat(fields.size(), is(0));
     }
 
+    @Test
+    public void testAddNullValueList()
+    {
+        HttpFields.Mutable fields = HttpFields.build();
+        assertThrows(IllegalArgumentException.class, () -> fields.add("name", (List<String>)null));
+        assertThat(fields.size(), is(0));
+        List<String> list = new ArrayList<>();
+        fields.add("name", list);
+        assertThat(fields.size(), is(0));
+
+        list.add(null);
+        fields.add("name", list);
+        assertThat(fields.size(), is(1));
+        assertThat(fields.get("name"), is(""));
+
+        list.add("");
+        fields.clear();
+        fields.add("name", list);
+        assertThat(fields.size(), is(1));
+        assertThat(fields.get("name"), is(", "));
```

Comment:
Such value is explicitly forbidden by the spec.

Reply:
I know.   But you said you wanted to accept such lists in your previous review???